jme3-examples/src/main/java/jme3test/bullet/TestIssue1120.java

```diff
@@ -65,7 +65,7 @@
  */
 public class TestIssue1120 extends SimpleApplication {
 
-    private static TestIssue1120 test;
+//    private static TestIssue1120 test;
```

Comment:
Please delete the commented-out code. Then this will be ready for integration.

Reply:
done